.changelog/4442.yml

```
@@ -0,0 +1,4 @@
+changes:
+- description: Add support in the `test-content` command for different api key file structure.
```

```suggestion
- description: Added support for different api key file structure in **test-content** command.
```
See our demisto-sdk changlog guide for more info: https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs.